Formula/mhonarc.rb

```diff
@@ -24,9 +24,6 @@ class Mhonarc < Formula
   end
 
   def install
-    # Handle the hardcoded binary script
-    inreplace "mhonarc", "#!/usr/bin/perl", "#!/usr/bin/env perl"
```

Comment:
Removed this as it should be automatically inreplaced with Homebrew perl shebang.